typescript/src/entities/product.test.ts

```diff
@@ -30,6 +30,19 @@ test('can fetch product from server', () => {
   return merchi.Product.get(1).then(product => expect(product.name).toBe(testName));
 });
 
+test('can specify options in request', () => {
+  const merchi = new Merchi();
+  const testName = 'S7qHUfV_dr5l';
+  const fetch = mockFetch(true, {'product': {'name': testName}}, 200);
+  (window as any).merchiBackendUri = 'http://override.example.com/';
```

Comment:
due to the merge of https://github.com/merchisdk/sdk/pull/198 

this line is not valid anymore. 

I will approve it, but I suggest you fix it here before you merge it